error.go

```diff
@@ -3,163 +3,171 @@
 package iproto
 
 // IPROTO error code constants, generated from
-// tarantool/src/box/errcode.h
+// tarantool-master/src/box/errcode.h
 type Error int
 
 const (
 	// "Unknown error"
 	ER_UNKNOWN Error = 0
-	// "Illegal parameters, %s"
+	// "%s" "" STRING
 	ER_ILLEGAL_PARAMS Error = 1
 	// "Failed to allocate %u bytes in %s for %s"
 	ER_MEMORY_ISSUE Error = 2
-	// "Duplicate key exists in unique index \"%s\" in space \"%s\" with old tuple - %s and new tuple - %s"
+	// nolint:lll
```

Comment:
The nolint will be removed then.

Reply:
Not in this case. Even if we delete the rest of it, this comment will be over 100 characters anyway.

Reply:
Let's remove this linter. All our files are generated and there is no much sense in that.

Reply:
Just removed adding this `nolint`. Cause, generated files ignored by linter.